DEVGUIDE.md

```diff
@@ -124,7 +124,7 @@ Running any of the above will build the latest changes and run tests against the
 
 ## Using your custom compiler to build this repository
 
-By removing all the subfolders called `Proto` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
+By removing all the subfolders called `Bootstrap` or `Proto` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
```

Comment:
Is it ever `Proto`, or should it be just `Boostrap` here?

Reply:
It is Bootstrap in the artifacts root and then Proto in bin/fsc.